Framework/PDFExporter/PDFExporter/Categories/UIKit/UITableView+PDFExporterDrawing.m

```diff
@@ -0,0 +1,125 @@
+//
+//  UITableView+Extension.m
+//  PDFExporter
+//
+//  Copyright © 2016 3Pillar Global. All rights reserved.
+//
+
+#import "UITableView+PDFExporterDrawing.h"
+#import "UIView+PDFExporterPageInformation.h"
+#import "CGGeometry+Additions.h"
+
+@implementation UITableView (PDFExporterExtension)
+
+- (void)drawViewWithinPageRect:(CGRect)rect {
+    CGRect drawingFrame = self.drawingFrame;
+    UIBezierPath *path = [UIBezierPath bezierPathWithRoundedRect:drawingFrame cornerRadius:self.layer.cornerRadius];
+    
+    [self drawBackgroundWithPath:path];
+    [self drawContentWithPath:path];
+    [self drawSubviewsWithPath:path withinPageRect:rect];
+    if (self.layer.borderWidth != 0) {
+        [self drawBorderWithPath:path];
+    }
+}
+
+- (void)drawSubviewsWithPath:(UIBezierPath *)path withinPageRect:(CGRect)rect {
+    CGPoint offset = CGPointZero;
+    for (CGFloat yCoordinate = CGRectGetMinY(rect);
+         yCoordinate < CGRectGetMaxY(rect);
+         yCoordinate += CGRectGetHeight(self.bounds)) {
+        offset.y = yCoordinate;
+        self.contentOffset = offset;
+        CGRect frame = self.frame;
+        [self layoutIfNeeded];
+        self.frame = frame;
```

Comment:
Do you expect ```self.frame``` to change after layout?  If it does change, it ends up being overridden back to the original frame anyways.  Why is this necessary?

Reply:
It is not needed anymore.